### DISCUSSION TITLE
Centered text in Entry widget ?

### DISCUSSION THREAD

**Discussion by Lennart1978** [fyne-io/fyne#4316] [Q&A] [answered]
Is there a possibility to center the text inside an Entry widget ? It's always on the left side and this does not look very good in some situations.

**Comment by andydotxyz** [comment, marked as answer]
There is no API for that.
If you can share details of the "some situations" perhaps we can understand the use-case better and consider it...

**Comment by Lennart1978** [comment]
Thanks for your very fast reply and sorry for my bad English.
Here is a screenshot of my app.
![screenshot_wol](https://github.com/fyne-io/fyne/assets/119766037/e139da61-b245-4f12-8cce-35b31ab0fe70)

There are 2 entry widgets and the text is on the left side. It would be nice if there would be a possibilty to center the text of the entrys.

**Reply by andydotxyz** [reply]
Are you sure? If you do that and have users type then the position of the text would change on every keystroke.
What is the reason in this situation to not use the leading (left) edge alignment (default)?

**Reply by Lennart1978** [reply]
That's not a bad thing, it would just be a little nicer visually.
